src/parser/pegen.ts

```diff
@@ -1762,30 +1752,58 @@ function _set_name_context(e: Name, ctx: expr_context) {
 //     return _Py_Starred(_PyPegen_set_expr_context(p, e->v.Starred.value, ctx), ctx, EXTRA_EXPR(e, e));
 // }
 
-/* Creates an `expr_ty` equivalent to `expr` but with `ctx` as context */
+/* Creates an asdl_seq* where all the elements have been changed to have ctx as context */
+function _set_seq_context(p: Parser, seq: expr[], ctx: expr_context) {
```

Comment:
we should add return types here

Reply:
Or did you omit on purpose?

Reply:
Will do. Sometimes I forget because it doesn't complain